benchmarks/multiplatform/benchmarks/src/commonMain/kotlin/MeasureComposable.kt

```diff
@@ -124,6 +127,20 @@ suspend fun measureComposable(
         )
     } finally {
         scene.close()
+        surface.close()
         runGC() // cleanup for next benchmarks
     }
 }
+
+private val pictureRecorder = PictureRecorder()
+
+@OptIn(InternalComposeUiApi::class)
+fun ComposeScene.mimicSkikoRender(surface: Surface, time: Long, width: Int, height: Int) {
```

Comment:
imo: requires BIG TODO to keep it in sync with skiko implementation, because I'm not sure that it won't be changed there

Reply:
Added a comment here

Without reusing the whole renderer from Skiko (which realistically we won't do without a good reason), we should keep it sync manually.

When we merge it, I will add a comment to Skiko:
```
Note, if you change the rendering logic a lot, notify about this in [this issue](new created issue), because [benchmarks project](link to this function) depends on it
```

Reply:
https://github.com/JetBrains/skiko/pull/1035